### ISSUE TITLE
Error when Trying to Convert DTOs

### ISSUE THREAD

**Issue by fabich** [CodySchrank/MTT#18]
Hi

I receive the following error:
```
The "ConvertMain" task failed unexpectedly.
System.ArgumentException: For parsing, C# DTO's must use curly braces on the next line
in AllocationRuleDTO.cs
"        public string Routenumber { get; set; }"
```

my DTO:

```
    public abstract class AllocationRuleBaseDTO : VersionedElementBaseDTO
    {
        public int FkProductgenerickey { get; set; }
        public string Dep { get; set; }
        public string Flightnumber { get; set; }
        public string Routenumber { get; set; }
        public int? FkDepregiongenerickey { get; set; }
        public string Dest { get; set; }
        public int? FkDestregiongenerickey { get; set; }
        public TimeSpan? Starttime { get; set; }
        public TimeSpan? Endtime { get; set; }
        public TimeSpan? DurationStarttime { get; set; }
        public TimeSpan? DurationEndtime { get; set; }
        public List<int> FkAircraftregistrations { get; set; }
        public List<int> FkAircraftTypes { get; set; }
        public string Carrier { get; set; }
        public string Owner { get; set; }
        public DateTime? SpecialStartdate { get; set; }
        public DateTime? SpecialEnddate { get; set; }
        public bool? IsReturnRule { get; set; }
    }
```

Is the problem related to TimeSpan?

**Comment by CodySchrank** [comment]
Timespan would get converted to any.  I'm not sure why that dto would fail, let me try to run it.

**Comment by CodySchrank** [comment]
Its because I have a check for `line.contains("enum")` and your line is `public string Routenumber { get; set; }` i.e. Rout - enum - ber!  Somehow that has never come up.  Let me change them all to strict checks and it will work for ya.

**Comment by CodySchrank** [comment]
fixed in 0.5.6

**Comment by fabich** [comment]
thank you @CodySchrank!
